### DISCUSSION TITLE
Happier with More Control?

### DISCUSSION THREAD

**Discussion by ngcthuong** [lllyasviel/ControlNet#227] [Ideas]
My experience when playing around with the guidance scale of ControlNet seems to indicate that increasing of guidance scale

- Image tends to get higher contrast
- Also smoother, fewer details in some controls
- Image looks "happier". Yeah, sound weird. --> 'properly because the input image has a smile'. 
- Hair tends to preserve more at higher control as well. 

Does any experience the same?

Control info: 
Keep resolution: True, Image resolution: 512, Strength: 1, Guess mode: False, Ddim steps: 20, Scale: 5, Seed: 816092623, Eta: 0.0
Prompt: , best quality, extremely detailed
Negative prompt: longbody, lowres, bad anatomy, bad hands, missing fingers, extra digit, fewer digits, cropped, worst quality, low quality

![image](https://user-images.githubusercontent.com/6524339/222985271-74a79ae1-91ee-4f04-b8f5-454e0618ae62.png)
![image](https://user-images.githubusercontent.com/6524339/222985278-67a57bde-f2f6-42a6-9fe9-2a976743fc16.png)
![image](https://user-images.githubusercontent.com/6524339/222985389-bf2d493d-03bf-485a-8b1f-0c34c9c6fff2.png)
![image](https://user-images.githubusercontent.com/6524339/222985293-9ff1f1ec-815c-48e4-9705-5c1be49107c5.png)

**Comment by geroldmeisinger** [comment]
increasing guidance scale in Stable Diffusion is generally known to produce high-contrast over-satured images. I don't think it's a problem with controlnet.
